nx-X11/programs/Imakefile

```diff
@@ -23,156 +18,15 @@ XCOMM $XFree86: xc/programs/Imakefile,v 3.54 2003/04/14 20:37:16 herrb Exp $
 #define IHaveSubdirs
 #define PassCDebugFlags CDEBUGFLAGS="$(CDEBUGFLAGS)"
 
-#if BuildServer || UseRgbTxt
-       RGBSRCDIR = rgb
-#endif
 #if BuildServer
         XSSRCDIR = Xserver
 #endif
-#if BuildXkbcomp
-      XKBCOMPDIR = xkbcomp
-#endif
-#ifndef Win32Architecture
```

Comment:
Be careful here. Note that from this line on, #if[n]defs are doubly-nested. Where does this #ifndef Win32Architecture stop?